docs/components/board/_index.md

```diff
@@ -161,7 +161,7 @@ For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/
 my_board = Board.from_robot(robot=robot, name="my_board")
 
 # Get the Analog "my_example_analog_pin".
-analog = await my_board.analog_by_name(name="my_example_analog")
+analog = await my_board.analog_reader_by_name(name="my_example_analog")
```

Comment:
this was changed by https://github.com/viamrobotics/docs/commit/2b889fe6f7a6a9db44f6634845f0080c2810a162 where I believe they were trying to get rid of `reader`. But I don't see `analog_by_name` in the sdk. @randhid can you let us know if that's an issue in the SDK?

Reply:
Specifically this should be changed to match python
https://python.viam.dev/autoapi/viam/components/board/index.html#viam.components.board.Board.analog_by_name